### ISSUE TITLE
Running interpreter on an experiment should set `this._inExperiment = true` if no `return` is run

### ISSUE THREAD

**Issue by dobesv** [rawls238/PlanOut.js#79]
I believe the PlanOut language expects that if you don't `return false` then the experiment is considered active.

Currently the interpreter only sets `_inExperiment = true` if you explicitly `return true` in the script.

**Comment by rawls238** [comment]
Seems reasonable, feel free to PR

**Comment by eytan** [comment]
Yes, at some point we eliminated the magic variable, in experiment. Returning false is the way folks should be disabling logging.
